main.js

```diff
@@ -160,6 +160,17 @@ async function initDatabase(db){
 		const articlesJson = JSON.parse(await readFile('./data/articles.json', 'utf-8'))
 		await db.collection("BBY-6_articles").insertMany(articlesJson);
 	}
+
+	const kits = await db.collection("BBY-6_kit-templates").find({}).toArray();
+	if(kits.length === 0 ){
```

Comment:
The `.toArray().length === 0` thing has been bugging me. This is seriously the nittiest of picks, but a maybe-better way to do this might be:

```js
if (! await db.collection("BBY-6_kit-templates").find({}).hasNext()) {

}
```
Since you're not using the data the `kits` cursor returns, anyway.
Theoretically this is better because it doesn't load any data from the database into RAM.